Fix credit card input from being cleared in OCS when save payment method checkbox is selected (#4964)
woocommerce/woocommerce-gateway-stripe: changelog.txt

* Dev - Use WC_STRIPE_PLUGIN_PATH constant instead of __DIR__ for more reliable file path resolution
* Dev - Automate release note creation PR
* Dev - Improve the pre-push hook

> * Fix - Prevent credit card input from being cleared when toggling save payment method checkbox in classic checkout with Optimized Checkout enabled

Reviewer: Nit: This is really long. I think we can simplify it a bit. Also, I checked that it affects other payment methods that have input fields, not just cards.
```suggestion
* Fix - Keep payment method details when toggling save payment method checkbox in classic checkout with Optimized Checkout enabled
```
